php/php.blade/src/org/netbeans/modules/php/blade/editor/HyperlinkProviderImpl.java

```diff
@@ -134,19 +134,21 @@ public int[] getHyperlinkSpan(Document doc, int offset, HyperlinkType type) {
 
                         for (FileObject includedFile : includedFiles) {
                             goToFile = includedFile;
-                            tooltipText = "Blade Template File : <b>" + viewPath
-                                    + "</b><br><br><i style='margin-left:20px;'>" + identifiableText + "</i>";
+                            tooltipText = "Blade Template File : <b>" + viewPath // NOI18N
```

Comment:
No. we should consider internationalization(i18n) for this part("Blade Template File :")